mlrun/utils/helpers.py

```diff
@@ -136,6 +136,15 @@ def verify_field_regex(field_name, field_value, patterns):
             )
 
 
+# Verifying that a field input is of the expected type. If not the method raises a detailed MLRunInvalidArgumentError
+def verify_field_of_type(field_name: str, field_value, expected_type: type):
```

Comment:
In case you follow my 2 other comments, this can be either private or at least not imported in sparkjob.